spec/Section 4 -- Introspection.md

```diff
@@ -16,7 +16,7 @@ type User {
 }
 ```
 
-The operation
+The request:
```

Comment:
I feel that calling this a request explicitly may be confusing - technically it's sufficient, but it doesn't provide space for providing variables/etc which are commonly part of a request, making it harder for the reader to reason about what a request really is. Perhaps change it to something like: `A request containing the following operation:`?

Reply:
Fixed in 324a6bfd898b1a6572bc810c3ce0a5c65e5b721e